trillian/ctfe/configpb/config.proto

```diff
@@ -76,6 +76,9 @@ message LogConfig {
   // checked against the current time during the validation of submissions.
   // This will cause expired certificates to be rejected.
   bool reject_expired = 6;
+  // If reject_non_expired is true then CTFE rejects certificates that are
+  // within their validity period with respect to the current time.
+  bool reject_non_expired = 17;
```

Comment:
`ValidateLogConfig` could do with an update to reject configs with both `reject_expired` and `reject_non_expired` set to `true`.

Reply:
Done.